internals/search_test.py

```diff
@@ -55,14 +55,14 @@ def test_structured_query_terms__complex(self):
         [('-', 'field', '=', 'value', '')],
         search.TERM_RE.findall('-field=value '))
     self.assertEqual(
-        [('-', 'field', '>', 'value', '')],
-        search.TERM_RE.findall('-field>value '))
+        [('OR ', 'field', '>', 'value', '')],
```

Comment:
You have some `OR` unit test cases mixed into this negation PR.

Reply:
I also added `OR ` parsing in this PR.